sandbox/rocky/tf/algos/ddpg/ddpg.py

```diff
@@ -0,0 +1,233 @@
+from sandbox.rocky.tf.algos.network.actor_critic_net import ActorNet, CriticNet
+from sandbox.rocky.tf.algos.ddpg.replay_buffer import ReplayBuffer
+from rllab.algos.base import RLAlgorithm
+
+import tensorflow as tf
+from copy import copy
+import numpy as np
+import os
+
+
+class DDPG(RLAlgorithm):
+    def __init__(self,
+                 env,
+                 gamma=0.99,
+                 tau=0.001,
+                 observation_range=(-5, 5),
+                 action_range=(-1, 1),
```

Comment:
Assume that the algorithm always gets a normalized environment, where `action_range` is always in [-1, 1]. We need to implement normalization for gym Envs anyways as described in #64.